lib/src/main/java/com/diffplug/spotless/scala/ScalaFmtStep.java

```diff
@@ -92,11 +92,27 @@ FormatterFunc createFormat() throws Exception {
 
 				Class<?> optionCls = classLoader.loadClass("scala.Option");
 				Class<?> configCls = classLoader.loadClass("org.scalafmt.config.Config");
-				Method fromHocon = configCls.getMethod("fromHocon", String.class, optionCls);
-				Object fromHoconEmptyPath = configCls.getMethod("fromHocon$default$2").invoke(null);
 
-				String configStr = new String(Files.readAllBytes(file.toPath()), StandardCharsets.UTF_8);
-				Object either = fromHocon.invoke(null, configStr, fromHoconEmptyPath);
+				Object either;
+
+				try {
+					// scalafmt >= v0.7.0-RC1
+					Method fromHocon = configCls.getMethod("fromHoconString", String.class, optionCls);
+					Object fromHoconEmptyPath = configCls.getMethod("fromHoconString$default$2").invoke(null);
+
+					String configStr = new String(Files.readAllBytes(file.toPath()), StandardCharsets.UTF_8);
+
+					Object configured = fromHocon.invoke(null, configStr, fromHoconEmptyPath);
+					either = invokeNoArg(configured, "toEither");
+				} catch (Exception e) {
```

Comment:
Hmm... is it possible to reduce the scope from `Exception` to a specific subclass?